package/winget/zurawiki.gptcommit.installer.yaml

```
@@ -0,0 +1,21 @@
+PackageIdentifier: zurawiki.gptcommit
+PackageVersion: <VERSION>
+ReleaseDate: <DATE>
+InstallerType: zip
+Installers:
+  - Architecture: x64
+    InstallerUrl: https://github.com/zurawiki/gptcommit/releases/download/v0.5.6/gptcommit-x86_64-pc-windows-msvc.zip
```

We should avoid hard-coding the version numbers here. Can we use the `<VERSION>` pattern here and below?
```suggestion
    InstallerUrl: https://github.com/zurawiki/gptcommit/releases/download/v<VERSION>gptcommit-x86_64-pc-windows-msvc.zip
```

---

Yes, it can be replaced.